source/codegen/templates/service_helpers.mako

```diff
@@ -201,10 +201,11 @@ ${initialize_standard_input_param(function_name, parameter)}\
   field_name = common_helpers.camel_to_snake(parameter["name"])
   request_snippet = f'request->{field_name}()'
   c_type = parameter['type']
+  grpc_type = parameter.get('grpc_type', None)
   c_type_pointer = c_type.replace('[]','*')
   c_type_underlying_type = common_helpers.get_underlying_type_name(c_type)
 %>\
-% if c_type == 'ViConstString':
+% if c_type in ['ViConstString', 'const char*']:
```

Comment:
Eventually I want to handle this a different way (at least for DAQ). But I think it's better to wait until we have more types to work with (maybe as part of the acquisition features).

Reply:
Yeah, I agree. I think we can wait to refine this as we get further into the DAQ features.